### DISCUSSION TITLE
CMT2300A not connected

### DISCUSSION THREAD

**Discussion by eRacoon42** [tbnobody/OpenDTU#952] [Q&A]
Hi guys,
very new to the project and trying to get my HMS-2000-4T working with the OpenDTU but had no luck so far.

I have connected the E49 900M20S like this [here](https://i.ibb.co/0VFnHqg/overview.png).

Next I uploaded the pin profile with the matching pin layout which looks like [this](https://i.ibb.co/S6G0FH0/2023-05-23-20-56-47-Open-DTU.png).

Not sure but looks fine for me, but the system status shows always not connected for the CMT2300A device like shown [here](https://i.ibb.co/bL3H2Hk/2023-05-23-20-58-16-Open-DTU.png).

Any debugging options I can use? Console is showing nothing?

Please some help for a noob like me. 
Thanks. :)

**Comment by tbnobody** [comment]
A picture of the really connected wires would help

**Comment by eRacoon42** [comment]
Hi!
[Here ](https://i.ibb.co/X3wph1D/20230523-220421.jpg)it is... not the best soldering skills maybe but all points have contact and nothing is bridged.
It is  the ESP32 chip like shown in the diagram with the pin layout from the first post.

**Comment by tbnobody** [comment]
Just to doublecheck that it's not a software issue I've applied the pin config here:
![image](https://github.com/tbnobody/OpenDTU/assets/1753222/255e3397-0629-495f-9c58-9d065ccdf427)


And it's working:
![image](https://github.com/tbnobody/OpenDTU/assets/1753222/a6a068ff-c80c-4eb1-93a9-ed78a4a18abc)


Sorry. changed it.... description is correct.... 8 = +3.3V and 9 = GND

Did you double check if the material on the foam is conductive?

**Comment by eRacoon42** [comment]
Thanks for the ideas, I did a resoldering of the chip and now it says "connected", looks like there was something not 100% connected.
But the modul still can't find my inverter... the box is only 1m away from the inverter, so range should be no problem.

Do I need to change anything else like db, frequency etc. for the HMS-2000-4T or is anything on default okay?
I only added my inverter serial number, it was auto detected to be a HMS serie and added the pin layout.
Or is there anything directly on the inverter that need to be enabled?

Next I would try to swap the E49 module, maybe I damaged the first one, I have two more as spare here.

Console looks like [this](https://i.ibb.co/0nqJRzb/2023-05-24-00-17-46-Open-DTU.png) in an endless loop.

**Comment by eRacoon42** [comment]
Powered on the device again in the morning and now it is working fine.
Maybe it was not the best idea to try this in the middle of the night?
I guess it is normal that the inverter is offline at nights?

**Comment by tbnobody** [comment]
>  I guess it is normal that the inverter is offline at nights?

Yes, it will be supplied by the solar panels only
